src/prefect/cli/flow_run.py

```diff
@@ -16,7 +16,7 @@
 from prefect.cli._types import PrefectTyper
 from prefect.cli._utilities import exit_with_error, exit_with_success
 from prefect.cli.root import app
-from prefect.client.orchestration import get_client
+from prefect import get_client
```

Comment:
For all internal uses, we should probably still use the fully-specified `prefect.client.orchestration`.

Reply:
Ah make sense. I will just update the doc for users.